prow/config/config.yaml

```diff
@@ -80,6 +80,81 @@ postsubmits:
         - image: alpine
           command:
             - /bin/printenv
+
+  nephio-project/free5gc-operator:
+  - name: build-push-image-free5gc-operator-release-conf
+    cluster: default
+    always_run: true
+    branches:
+    - "^v.*$"
+    annotations:
+      description: Build and Push Image to DockerHub when release tag is created
+    decorate: true
+    decoration_config:
+      censor_secrets: true
+    max_concurrency: 0
+    spec:
+      containers:
+      - name: kaniko
+        image: gcr.io/kaniko-project/executor:debug
+        command:
+        - "/bin/sh"
+        - "-c"
+        - |
+          executor --context=${PWD}/free5gc-operator \
+            --dockerfile=Dockerfile --destination=nephio/free5gc-operator:${PULL_BASE_REF}
+        volumeMounts:
+          - name: kaniko-secret
+            mountPath: /kaniko/.docker/
+        resources:
+          requests:
+            cpu: 2
+            memory: 2Gi
```

Comment:
No a problem / blocker. But 2 vCPUs and 2 GB memory seems like a LOT for the operator...